packages/clerk-js/src/core/resources/Organization.ts

```diff
@@ -109,11 +109,16 @@ export class Organization extends BaseResource implements OrganizationResource {
   };
 
   getRoles = async (getRolesParams?: GetRolesParams) => {
-    return await BaseResource._fetch({
-      path: `/organizations/${this.id}/roles`,
-      method: 'GET',
-      search: convertPageToOffset(getRolesParams) as any,
-    }).then(res => {
+    return await BaseResource._fetch(
```

Comment:
🙃 i think there is no need to `return await` in an async function. this PR didn't change the existing codebase but i think we should jest return the promise. This is a suggestion for the future.